source/includes/language-compatibility-table-php.rst

```diff
@@ -0,0 +1,158 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large
+
+   * - PHP Driver Versions
+     - PHP 8.4
+     - PHP 8.3
+     - PHP 8.2
+     - PHP 8.1
+     - PHP 8.0
+     - PHP 7.4
+     - PHP 7.3
+
+   * - ext + lib 1.20
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+
+   * - ext + lib 1.17 to 1.19
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     -
+
+   * - ext + lib 1.16
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+
+   * - ext + lib 1.15 [#PHP1.15-version-parity]_
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - ext 1.14 + lib 1.13
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - ext 1.13 + lib 1.12
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - ext 1.12 + lib 1.11
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - ext 1.11 + lib 1.10
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - ext 1.10 + lib 1.9
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - ext 1.9 + lib 1.8
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - ext 1.8 + lib 1.7
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+
+   * - ext 1.7 + lib 1.6
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+
+   * - ext 1.6 + lib 1.5
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+
+   * - ext 1.5 + lib 1.4
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+
+   * - ext 1.4 + lib 1.3
+     -
+     -
+     -
+     -
+     -
+     -
+     - ✓
+
+   * - ext 1.3 + lib 1.2
+     -
+     -
+     -
+     -
+     -
+     -
+     - ✓
```

Comment:
S: Can you check in with the PHP team or Andreas about whether we can remove some of these old versions from this table? Right now, the table is very long as it includes all of these old php driver versions

Reply:
Hi @alcaeus, are we able to remove any of the older versions of the php driver from the compatibility table? Thanks!

Reply:
These old versions definitely aren't relevant anymore. I think we can drop anything older than ext 1.14 and lib 1.13, as those were released two years ago.

Reply:
According to Slack conversation, will remove older than ext + lib 1.15